tests/loaders_test.py

```diff
@@ -15,7 +15,9 @@
 from elastalert.loaders import RulesLoader
 from elastalert.util import EAException
 
-test_config = {'rules_folder': './empty_folder_test',
```

Comment:
Tests were failing in Visual Studio Code because it was running tests relative to the root of the project, instead of the `tests` directory.